helm/values.local.yaml

```diff
@@ -10,3 +12,15 @@ fluentbit:
 cronJob:
   # Overrides netapp-file-backup which is only valid in OpenShift.
   storageClassName: hostpath
+
+ingress:
+  enabled: false
+
+mongodb:
+  auth:
+    rootPassword: 'rootpassword'
+    replicaSetKey: 'replicasetpassword'
+    password: 'password'
+  extraEnvVars:
+    - name: EXPERIMENTAL_DOCKER_DESKTOP_FORCE_QEMU
```

Comment:
is this what made the pods come up without errors?

Reply:
That's right. The mongodb pods were failing to start up because mongodb doesn't release images for arm64 architecture. This env variable seems to make it so Docker Desktop uses the QEMU emulator instead of Rosetta which for whatever reason makes it possible to run the image built for amd64 on arm64. I got this information from here: https://github.com/docker/for-mac/issues/7137